templates/individual_response/how.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'question.html' %}
+
+{% set save_on_signout = true %}
+{% set question = content.block.question %}
+{% set continue_button_text = _("Continue") %}
+
+{% block after_submit_button_content %}
+    {% if is_post_confirm_page %}
+      <p class='u-mt-xl'>{{ _("To request a census in a different format or for further help, please <a href='{contact_us_url}'>contact us</a>".format(contact_us_url=contact_us_url)) }}</p>
+    {% endif %}
+{% endblock %}
```

Comment:
As discussed - we should create an individual response `question.html` and this should extend that to add the extra after submit button content, removing the need for the `is_post_confirm_page` boolean.